src/net.cpp

```diff
@@ -2492,6 +2492,9 @@ void DumpBanlist()
     CNode::GetBanned(banmap);
     bandb.Write(banmap);
 
+    // Notify UI about changed banlist
+    uiInterface.BannedListChanged();
```

Comment:
Hmm... emit the new ui signal from `DumpBanlist()` would result in multiple calling of `banTableModel->refresh();`. I'd prefer keeping this outside of the core classes (keep the call in the rpcnet.cpp methods).
Or if you like to move it, you need to take care `RPCConsole::banSelectedNode(int bantime)`, etc. to avoid multiple calls to the same method.